Makefile

```diff
@@ -15,21 +15,22 @@ all:
 	docker build -t kube-spawn-build:$(DOCKER_TAG) -f Dockerfile.build .
 	mkdir -p $(GOCACHEDIR)
 	docker run --rm -ti \
-		-v `pwd`:/go/src/github.com/kinvolk/kube-spawn:Z \
+		-v `pwd`:/usr/src/kube-spawn:Z \
 		-v $(GOCACHEDIR):/tmp/.cache:Z \
 		--user $(UID):$(UID) \
 		kube-spawn-build
 else
 all:
-	go build -o kube-spawn \
-		-ldflags "-X main.version=$(VERSION)" \
+	export GO111MODULE=on
+	go mod download
+	go build -ldflags "-X main.version=$(VERSION)" \
 		./cmd/kube-spawn
```

Comment:
Sorry, I was not specific.
The current Makefile still does not set `GO111MODULE` correctly.

In Makefile, you need to specify env variables either in the same line you want to run, or in one of the multiple lines separated by `;`.

For example you can do either

```
        go mod download
        GO111MODULE=on go build
```

or

```
        export GO111MODULE=on; \
        go mod download; \
        go build
```
See also https://stackoverflow.com/questions/42993169/setting-environment-variable-in-makefile-and-reading-it-from-python-script-does

Reply:
Yes, I have this one fixed already, just didn't push it yet. Testing takes time...